worker/include/RTC/RTCP/CompoundPacket.hpp

```diff
@@ -60,8 +68,8 @@ namespace RTC
 			// Use `CompoundPacket::ReturnIntoPool()` instead
```

Comment:
With `ReturnIntoPool` removed this comment is out of date now (maybe in other places as well).